reference/platform-app-yaml.md

```diff
@@ -289,13 +289,12 @@ Possible hooks are:
     application has not been deployed yet.
 -   **deploy**: We run deploy hooks after your application has been
     deployed and started. You can access other services at this stage
-    (MySQL, Solr, Redis...).
+    (MySQL, Solr, Redis...). However, the disk where the application lives is read-only at this point.
 
 > **Note**
-> The "home" directory is /app while your application will be
-mounted in /app/public (by default: you can define this yourself in your
-.platform.app.yaml file) - so you might want to `cd public` before
-running hooks.
+> Your application will always be mounted under `/app`, and whatever subdirectory
+you specified in the `web/locations/root` key above.  For example, `/app/web`. You may need
```

Comment:
Well, it's the application that is mounted there. ie, the directory tree starting from .platform.app.yaml.  That it's also $HOME when sshing in is not relevant here, I'd think.